### PR TITLE
fix two defects in adjust-cinder-backend.yml

### PR DESCRIPTION
**Pull request by chengtcli**
1. create cinder volume type failed, because cinder service is not started. Fixed it by waiting for 30s for cinder service restarted.
2. `ceph.bcache_ssd_device` is defined as ceph_osds_hybrid group variable, while adjust-cinder-backend.yml is against controller nodes. So `ceph.bcache_ssd_device` is always undefined.

### CODE REVIEW

**Comment by chengtcli** [comment]
Have tested on hybrid env.

After adjust-cinder-backend.yml,
1. cinder.conf show right configuration
2. cinder type-list show CEPH_HYBRID
3. type of old volumes become CEPH_HYBRID
4. create a volume with `openstack volume create --size 10 open_hybrid`, it's type is CEPH_HYBRID
5. create a volume with `cinder create --volume_type CEPH_HYBRID --display_name hybrid1 10`, it's type is CEPH_HYBRID
6. all the volumes can be accessed in VM

**Comment by knandya** [comment]
@nirajdp These changes are tested in the dev environment and looks good.

**Comment by j2sol** [comment]
ok to test

File: playbooks/adjust-cinder-backend.yml [hunk 1]
```diff
@@ -47,6 +47,12 @@
       when: not cinder.enabled|default('False')|bool
       run_once: true
 
+    - fail: msg="Only one type of osds should be configured"
```
Comment (j2sol) [inline playbooks/adjust-cinder-backend.yml]: I didn't notice it before, but all of these tasks should have a name.

Comment (j2sol) [inline playbooks/adjust-cinder-backend.yml]: Also this looks like an unrelated change, please account for it in the commit message(s).

**Comment by knandya** [comment]
Per PR 2173, this playbook is outside of CI code path.

File: playbooks/adjust-cinder-backend.yml [hunk 2]
```diff
@@ -111,6 +117,11 @@
 
   - name: trigger stop all cinder services
     meta: flush_handlers
+    
+
+  - name: wait for cinder services down
+    shell: sleep 10s
+    when: set_cinder_host or update_cinder_backend
```
Comment (j2sol) [inline playbooks/adjust-cinder-backend.yml]: This feels fragile. Perhaps it would be better to use a `until` loop and check the ceph API status (maybe with a sensu plugin execution). That'd be safer and more accurate.

File: playbooks/adjust-cinder-backend.yml [hunk 3]
```diff
@@ -158,6 +169,10 @@
   - name: trigger restart cinder services
     meta: flush_handlers
 
+  - name: wait for cinder services up
+    shell: sleep 30s
+    when: set_default_volume_type or set_cinder_host or update_cinder_backend
```
Comment (j2sol) [inline playbooks/adjust-cinder-backend.yml]: Same here.

File: playbooks/adjust-cinder-backend.yml [hunk 4]
```diff
@@ -158,6 +169,10 @@
   - name: trigger restart cinder services
```
Comment (j2sol) [inline playbooks/adjust-cinder-backend.yml]: Unrelated, but meta calls don't need names.

File: playbooks/adjust-cinder-backend.yml (hunk 2 above)
Comment (j2sol) [inline playbooks/adjust-cinder-backend.yml]: No, I meant using the `command` module and directly executing a sensu plugin to check the status of the cinder service. Like https://github.com/blueboxgroup/ursula/blob/master/roles/percona-server/tasks/main.yml#L67-L72

File: playbooks/adjust-cinder-backend.yml (hunk 1 above)
Comment (chengtcli) [inline playbooks/adjust-cinder-backend.yml]: I tried check-cinder-service.sh under sensu plugin dir. it didn't work very well.
the sensu check can make sure service is up, while it can promise `cinder type-creat` works.
I use `cinder list` instead

File: playbooks/adjust-cinder-backend.yml (hunk 2 above)
Comment (chengtcli) [inline playbooks/adjust-cinder-backend.yml]: I tried check-cinder-service.sh under sensu plugin dir. it didn't work very well.
the sensu check can make sure service is up, while it can promise `cinder type-creat` works.
I use `cinder list` instead
